Mathlib/RingTheory/TensorProduct/Basic.lean

```diff
@@ -7,6 +7,8 @@ import Mathlib.Algebra.Algebra.RestrictScalars
 import Mathlib.GroupTheory.MonoidLocalization.Basic
 import Mathlib.LinearAlgebra.TensorProduct.Tower
 import Mathlib.RingTheory.Adjoin.Basic
+import Mathlib.RingTheory.Finiteness
```

Comment:
This basic file was recently split, so you can just move your lemma here to `Mathlib.RingTheory.TensorProduct.Finite`.